src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -1225,6 +1225,25 @@ object FunctionRegistry {
     ``fet`` is identical to the version implemented in `R <https://stat.ethz.ch/R-manual/R-devel/library/stats/html/fisher.test.html>`_ with default parameters (two-sided, alpha = 0.05, null hypothesis that the odds ratio equals 1).
     """,
     "a" -> "value for cell 1", "b" -> "value for cell 2", "c" -> "value for cell 3", "d" -> "value for cell 4")
+
+  register("binomTest", { (nSuccess: Int, n: Int, p: Double, alternative: String) => binomTest(nSuccess, n, p, alternative)
+  },
+    """
+    Performs a binomial test and returns the p-value.
```

Comment:
Returns the p-value of a `binomial test <https://en.wikipedia.org/wiki/Binomial_test>`__ of the null hypothesis that success has probability `p`, given `x` successes in `n` trials.